src/keycloak/chart/templates/istio-admin.yaml

```diff
@@ -56,4 +56,7 @@ spec:
             notNamespaces:
             - istio-tenant-gateway
             - istio-admin-gateway
+            {{- range .Values.additionalGatewayNamespaces }}
+            - {{ . }}
+            {{- end }}
```

Comment:
I think we could optionally enforce that this name starts with `istio-` as a small safeguard. But that does make the values override a bit more confusing/not just the namespace.

Reply:
Two options with this - helm's `fail` + a check on the string, or just add `istio-` to this template.

Reply:
I'd consider not doing anything as an option too, and just accepting the additional namespace.

Reply:
I'm in favor of a naming convention to set a standard for cluster cleanness/organization. If in the future we needed to accommodate other types of gateways, not just `istio-`, we could use a whitelist of allowed prefixes that's used in the validation.

Reply:
Updated to use a helm `fail` to enforce this prefix. We can always re-evaluate this decision but it should work for the only known consumer needing this.